pom.xml

```diff
@@ -335,6 +335,16 @@
         <enabled>true</enabled>
       </snapshots>
     </pluginRepository>
+    <!-- configure the EBR plug-in repo -->
+    <pluginRepository>
+      <id>ebr</id>
+      <url>https://repo.eclipse.org/content/repositories/ebr/</url>
```

Comment:
i'm especially interested in why not use http://projects.reficio.org/p2-maven-plugin/1.1.0/manual.html which seem to do the exact same thing but not have the downside of having to run multiple times ?